### PR TITLE
Prevent screen auto-lock while app is running

### PR DESCRIPTION
**Pull request by monchote**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="fill_parent"
```
Comment (capitancambio) [inline app/src/main/res/layout/activity_main.xml]: shouldn't this be match_parent?

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-    android:layout_height="fill_parent">
```
Comment (capitancambio) [inline app/src/main/res/layout/activity_main.xml:5]: Shouldn't this be match_parent as fill_parent is deprecated?

Comment (monchote) [inline app/src/main/res/layout/activity_main.xml:5]: 🖕

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent"
+    android:layout_width="match_parent"
```
Comment (capitancambio) [inline app/src/main/res/layout/activity_main.xml:4]: :+1:

**Pull request merged**